test/e2e/deferred/basic.go

```diff
@@ -0,0 +1,407 @@
+package e2e
```

Comment:
Do the test in `e2e/deferred/` actually run? I don't see the `Makefile` adjusted so that they run automatically.

Reply:
tight, need to hook them in. So far I'm running them explicitely (and exclusively)